README.md

```diff
@@ -11,6 +11,15 @@ The [sample application](https://github.com/IBM/sample.microservices.web-app) us
 
 ![Flow](images/architecture.png)
 
+## Flow
+
+1. Create Kubernetes service from IBM cloud.
+1. Deploy all microservices and web application also as a microservices into the Kubernetes cluster.
```

Comment:
not quite sure what this means - can you re-phrase? At a minimum, second "microservices" should not be plural